api_app/analyzers_manager/file_analyzers/file_info.py

```diff
@@ -33,6 +34,11 @@ def run(self):
         results["sha1"] = hashlib.sha1(binary).hexdigest()
         results["sha256"] = hashlib.sha256(binary).hexdigest()
         results["ssdeep"] = pydeep.hash_file(self.filepath).decode()
+        tlsh_hash = tlsh.hash(binary)
+        if tlsh_hash == "TNULL":
+            logger.warning(f"TLSH hash is NULL for {self.filepath}")
```

Comment:
could you also write this warning down to a separated key? in that way the user could understand when this happens directly from the GUI

Reply:
![image](https://user-images.githubusercontent.com/58381523/177046939-85c07419-4055-4009-b484-4ce2f044cf3a.png)
Isn't errors a list instead of a dictionary?

Reply:
my point was to add that string as an another key in the `report`, cause I considered it normal behavior.

If you think that it is not normal behavior and it may resemble an error, so you are right and we should add in the `errors` section. Do you have an example on when that "TNULL" happens in real life?

Reply:
https://github.com/trendmicro/tlsh#minimum-byte-stream-length

Reply:
I consider this a normal behavior. I mean, the program ran successfully. I would add a new key in the `report` so

Reply:
Alright. Will remove the check